### PR TITLE
Dont publish migration if migration exists

### PR DESCRIPTION
**Pull request by MortenDHansen**
As suggested with pr #644

### CODE REVIEW

File: src/AuditingServiceProvider.php [hunk 1]
```diff
@@ -52,11 +51,13 @@ private function registerPublishing()
                 __DIR__ . '/../config/audit.php' => base_path('config/audit.php'),
             ], 'config');
 
-            $this->publishes([
-                __DIR__ . '/../database/migrations/audits.stub' => database_path(
-                    sprintf('migrations/%s_create_audits_table.php', date('Y_m_d_His'))
-                ),
-            ], 'migrations');
+            if (!class_exists('CreateAuditsTable')) {
```
Comment (anteriovieira) [inline src/AuditingServiceProvider.php:54]: check if the file exists instead of the class.

**Review by anteriovieira — CHANGES_REQUESTED** [review]

**Comment by MortenDHansen** [comment]
> 

Yeah, but i don't think you actually need to use the anonymous migrations if you don't want to. If we expect all existing migrations to be created as a class and we dont change how we publish it, then the class check should be enough. Spatie seems use same approach: https://github.com/spatie/laravel-translation-loader/blob/main/src/TranslationServiceProvider.php

**Comment by anteriovieira** [comment]
Looks good to me! thanks

**Comment by sarukomine** [comment]
@MortenDHansen the change is not working for me, at least not working for Laravel Framework 9.9.0, I still can keep on publishing migration files. Maybe we should change the way to check if file exists instead of the class.

**Comment by MortenDHansen** [comment]
Yeah, maybe my brilliant simple fix was not as brilliant as i figured, and a bit too simple :/

**Pull request merged**